scripts/generate_ssl_cert

```diff
@@ -22,8 +22,12 @@ fi
 dnsnames=$((hostname -A; hostname -f) | sort | uniq)
```

Comment:
The man page says:
> If a machine has multiple network interfaces/addresses or is used in a mobile environment, then it may either have multiple FQDNs/domain names or none at all. Therefore avoid
       using hostname --fqdn, hostname --domain and dnsdomainname.  hostname --ip-address is subject to the same limitations so it should be avoided as well.
But I can see that `hostname -f` gives back a longer domain name that isn't returned by `hostname -A`, that one just returned the hostname of the host. So I guess we need this?

Reply:
That's right: from the manpage I would have thought that `hostname -A` on its own should be enough, but in practice `hostname -f` gives something that `hostname -A` doesn't, so I decided to use the union of the two sets of names (by doing `| sort | uniq`).